cernopendata_client/cli.py

```diff
@@ -96,7 +96,12 @@ def get_metadata(server, recid, doi, title, output_fields):
             )
             sys.exit(1)
     else:
-        output_json = record_json
+        output_json = record_json["metadata"]
+        for field in output_json["_files"]:
+            if "bucket" in field:
+                del field["bucket"]
+            if "version_id" in field:
+                del field["version_id"]
```

Comment:
Works well :+1: Can you please add simple assert into existing tests to say that these fields ("bucket" etc) should not be present in the output?

Reply:
okay 👍

Reply:
@tiborsimko There are two similar fields namely `_files` and `files` which have the same data. Should I remove these 2 keys from `files` as well?

Reply:
Indeed, good catch! You can therefore remove the whole `_files` field, let's keep only `files` and let's remove `bucket` and `version_id` from there. 

P.S. Writing tests has a nice side effect in spotting these things :wink:

Reply:
Yeah, I agree 😃